Sources/LanguageServerProtocol/CompletionItemKind.swift

```diff
@@ -15,40 +15,63 @@
 /// In LSP, this is an integer, so we don't use a closed set.
 public struct CompletionItemKind: RawRepresentable, Codable, Hashable {
 
-  public var rawValue: Int
+      public var rawValue: Int
 
-  public init(rawValue: Int) {
-    self.rawValue = rawValue
-  }
+      public init(rawValue: Int) { self.rawValue = rawValue }
 
-  // MARK: LSP completion kinds
+      // MARK: LSP completion kinds
 
-  // LSP 1 kinds, guaranteed to be supported by all clients.
-  public static let text: CompletionItemKind = CompletionItemKind(rawValue: 1)
-  public static let method: CompletionItemKind = CompletionItemKind(rawValue: 2)
-  public static let function: CompletionItemKind = CompletionItemKind(rawValue: 3)
-  public static let constructor: CompletionItemKind = CompletionItemKind(rawValue: 4)
-  public static let field: CompletionItemKind = CompletionItemKind(rawValue: 5)
-  public static let variable: CompletionItemKind = CompletionItemKind(rawValue: 6)
-  public static let `class`: CompletionItemKind = CompletionItemKind(rawValue: 7)
-  public static let interface: CompletionItemKind = CompletionItemKind(rawValue: 8)
-  public static let module: CompletionItemKind = CompletionItemKind(rawValue: 9)
-  public static let property: CompletionItemKind = CompletionItemKind(rawValue: 10)
-  public static let unit: CompletionItemKind = CompletionItemKind(rawValue: 11)
-  public static let value: CompletionItemKind = CompletionItemKind(rawValue: 12)
-  public static let `enum`: CompletionItemKind = CompletionItemKind(rawValue: 13)
-  public static let keyword: CompletionItemKind = CompletionItemKind(rawValue: 14)
-  public static let snippet: CompletionItemKind = CompletionItemKind(rawValue: 15)
-  public static let color: CompletionItemKind = CompletionItemKind(rawValue: 16)
-  public static let file: CompletionItemKind = CompletionItemKind(rawValue: 17)
-  public static let reference: CompletionItemKind = CompletionItemKind(rawValue: 18)
+      // LSP 1 kinds, guaranteed to be supported by all clients.
+      public static let text: CompletionItemKind = CompletionItemKind(
+            rawValue: 1)
```

Comment:
This particular wrapping looks pretty weird IMO, but it's not a dealbreaker.

Reply:
The intention is to adjust the rules to encourage the formatter to break before the `=` if it would not increase the overall number of lines.

Reply:
That grouping is a straightforward change that Austin or I can make tomorrow, and then push a new commit with those changes so we can see them isolated against this original format pass.